quetz/tasks/assertions.py

```diff
@@ -1,9 +1,9 @@
 def can_channel_synchronize(channel):
-    return channel.mirror_channel_url and (channel.mirror_mode == "mirror")
+    return channel.mirror_channel_urls and (channel.mirror_mode in ["mirror", "proxy"])
```

Comment:
this type of line is where I would really prefer mirror_mode to be an enum

Reply:
There already is an enum under `rest_models`. Done!